.eslintrc.json

```diff
@@ -25,6 +25,10 @@
         "@typescript-eslint/consistent-type-definitions": 0,
         "@typescript-eslint/explicit-function-return-type": 0,
         "@typescript-eslint/no-use-before-define": 0,
+        "class-methods-use-this": [
+            2,
+            { "exceptMethods": ["componentDidMount"] }
+        ],
```

Comment:
Had to add this override as eslint was complaining that `componentDidMount` was a class method without a reference to `this` in it following the refactor.